### PR TITLE
Account page functional + updated UI

### PR DESCRIPTION
**Pull request by abhaykshar**
image

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/services/SettingsService.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	override fun getChatNotifications(): Boolean {
+		return allowChatNotifications
+	}
+
+	companion object {
+		private const val DEFAULT_DISPLAY_SETTINGS_PATH = "settings.defaultdisplaytheme.subletr"
+		private const val DISPLAY_THEME_SETTINGS_PATH = "settings.displaytheme.subletr"
+		private const val CHAT_NOTIFICATIONS_SETTINGS_PATH = "settings.chatnotifications.subletr"
+	}
+}
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/services/SettingsService.kt:65]: Is there somewhere where you read the disk values when the app starts?
You can just do it on `init` I think?

File: android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt [hunk 2]
```diff
@@ -48,6 +109,18 @@ fun AccountPageView(
 		AccountPageUiState.Loading
 	).value,
 ) {
+	var nameExpanded by remember { mutableStateOf(false) }
+	var genderExpanded by remember { mutableStateOf(false) }
+	val initialPersonalInformation = remember {
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt]: Should probably use keyword `by` here

**Review by leeminkyu16 — APPROVED** [review]
Honestly, the 1000 line function isn't the prettiest thing but the actual screen is.
Let's Gamble, Try Merging

**Review by andr-dev — APPROVED** [review]
Nice!

**Pull request merged**